### ISSUE TITLE
Quiet mode on tusk.yml

### ISSUE THREAD

**Issue by WoLfulus** [rliebz/tusk#54]
**Is your feature request related to a problem? Please describe.**

There are some sensitive commands that shouldn't be logged to the console. But we're still interested in seeing logs from other commands.

**Describe the solution you'd like**

Provide a way permanently supress tusk output (inside root yaml, without having to pass -q) to supress logging and a way to override the output of specific commands inside `run` keys.

**Comment by rliebz** [comment]
Would something like this work for your use case?

```yaml
tasks:
  foo:
    run:
      command: my_command --password "$MY_SECRET_PASSWORD"
      echo: my_command --password "*****"
```

Where the `echo` clause would take an arbitrary string that would replace the normal command output.

The reason I'm leaning toward replacing the command text rather than disabling it is so that users would be encouraged to document an output that describes what's going on, rather than losing information.

**Comment by WoLfulus** [comment]
Yeah, that should work just fine! :)

**Comment by smyrman** [comment]
With the suggested solution, probably expand variables in `echo` as well 😉 
```yaml
tasks:
  foo:
    run:
      command: my_command $MY_NON_SECERT_OPTIONS --password "$MY_SECRET_PASSWORD"
      echo: my_command $MY_NON_SECERT_OPTIONS --password "*****"
```

Another option I want to mention, could be to pass the environment variable through an _option_ as secret or sensored:

```yaml
tasks:
  foo:
    options:
      my_password:
        type: string
        usage: "secret password"
        default: "$MY_SECRET_PASSWORD"  
        required: true
        secret: true
    run:
      command: my_command --password $my_passwoed
```

Then, when ever the option is used, the command that is printed would resolve sensored variables to `****` instead of the actual value without having to set a custom echo string.

The reason I mention this is, because it would fit well with my suggestion on #43; if options could be prompted for and passed in interactively:

```
```yaml
tasks:
  foo:
    options:
      my_password:
        type: string
        usage: "secret password"
        default: "$MY_SECRET_PASSWORD"  
        promt: "Enter password"
        required: true
       secret: true # use a prompt type "password" over "string" in survey.
    run:
      command: my_command --password $my_passwoed
```

**Comment by WoLfulus** [comment]
don't discard `echo` though, because we'd use them to describe the steps taken for each command instead of outputting the command itself

**Comment by rliebz** [comment]
Sorry for the delay--to solve this I had to make a technically breaking change, and wanted to batch it with a couple other things. This is now available with the `print` directive in `v0.5.0`: https://rliebz.github.io/tusk/spec/#print

Marking variables as secret is an idea I'm still interested in, it's just a little trickier to implement based on the way variable interpolation happens right now. I'm going to close this issue and open another to address that specifically.
